### PR TITLE
lastfm init

### PR DESCRIPTION
**Pull request by vialikanau-aliaksei**

### CODE REVIEW

**Review by NikitaSedyx — APPROVED** [review]
do you use jsdoc or is it just annotations?

**Comment by vialikanau-aliaksei** [comment]
just for memo

**Comment by vialikanau-aliaksei** [comment]
https://github.com/vialikanau-aliaksei/vialikanau-aliaksei.github.io/pull/6


Ад: Mikita Siadykh
Адпраўлена:27 красавіка 2017 16:11
Каму: vialikanau-aliaksei/vialikanau-aliaksei.github.io
Копія: vialikanau-aliaksei; Author
Тэма: Re: [vialikanau-aliaksei/vialikanau-aliaksei.github.io] lastfm init(#6)

@NikitaSedyx approved this pull request.
do you use jsdoc or is it just annotations?

File: lastfm/js/test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+var assert = chai.assert;
+
+describe("getRequest.doGet", function () {
+  it("(getRequest.doGet(api.searchArtist('Cher'), function(){}) exec onload", function (done) {
+    http.doGet(api.searchArtist('Cher'), function () {
```
Comment (NikitaSedyx) [inline lastfm/js/test.js:5]: actually sinon is required to spy on methods
common practise for mocha: mocha + chai + sinon

**Pull request merged**